components/tools/OmeroPy/src/omero/util/sessions.py

```diff
@@ -347,8 +348,13 @@ def create(self, name, pasw, props, new=True, set_current=True, sudo=None):
             principal.name = name
             principal.group = props.get("omero.group", None)
             principal.eventType = "User"
+            try:
+                cs = sf.getConfigService()
+                idleTimeout = long(cs.getConfigValue('omero.sessions.timeout'))
+            except:
+                idleTimeout = 0L
```

Comment:
Improved by retrieving the `timeToIdle` from the initial (sudoer) session instead of querying the server again.